ql/src/java/org/apache/hadoop/hive/ql/parse/LoadSemanticAnalyzer.java

```diff
@@ -181,7 +184,16 @@ private List<FileStatus> applyConstraintsAndGetFiles(URI fromURI, Table table) t
             ErrorMsg.INVALID_PATH.getMsg(), fromTree,
             "No files matching path " + fromURI));
       }
-
+    } catch (IOException e) {
+      throw new SemanticException(ASTErrorUtils.getMsg(
+              ErrorMsg.INVALID_PATH.getMsg(), fromTree), e);
+    }
+    return srcs;
+  }
+    private List<FileStatus> applyConstraintsAndGetFiles(URI fromURI, Table table, FileStatus[] srcs)
+            throws SemanticException {
+    try {
+      FileSystem fileSystem = FileSystem.get(fromURI, conf);
```

Comment:
why do you need `fileSystem` here?

Reply:
Down the line in the applyConstraintsAndGetFiles, filesystem is getting used in "AcidUtils.validateAcidFiles(table, srcs, fileSystem);"

Reply:
@deniskuzZ Can you look at the changes and review it ?

Reply:
Move this to above the line where it is used.

Reply:
This comment  is update and resolved